teamA/lib/Views/login_page.dart

```diff
@@ -37,6 +38,7 @@ class _LoginScreenState extends State<LoginScreen>
 {
   final TextEditingController _usernameController = TextEditingController();
   final TextEditingController _passwordController = TextEditingController();
+  final TextEditingController _moodleURL = TextEditingController();
```

Comment:
I would name this moodleURLController just so it's obvious that's the variable type, as well as to follow the existing convention.